spec/fixtures/rails_app/Gemfile

```diff
@@ -10,6 +10,8 @@ gem 'rails'
 gem 'sqlite3'
 # Use Puma as the app server
 gem 'puma'
+
+gem 'devise', github: 'heartcombo/devise', branch: 'main'
```

Comment:
@waiting-for-dev I thought that making the test suite run against the main branch of `devise` can help us catch any breaking change early on.

this goes hand in hand with `spec.add_dependency 'devise', '>= 4.0'`

otherwise it won't work @waiting-for-dev